CMakeLists.txt

```diff
@@ -24,14 +27,18 @@ set(ENABLE_TESTS On CACHE BOOL "")
 set(ENABLE_BENCHMARKS On CACHE BOOL "")
 set(ENABLE_GMOCK On CACHE BOOL "")
 set(ENABLE_DOCS Off CACHE BOOL "")
-set(ENABLE_WARNINGS_AS_ERRORS Off CACHE BOOL "")
 option(ENABLE_EXAMPLES "Build Umpire examples" On)
 option(ENABLE_LOGGING "Build Umpire with Logging enabled" On)
 option(ENABLE_SLIC "Build Umpire with SLIC logging" Off)
 option(ENABLE_STATISTICS "Track statistics for allocations and operations" Off)
 option(ENABLE_COVERAGE "Enable code coverage (with GCC)" Off)
-option(ENABLE_TOOLS "Enable Umpire development tools" Off)
 option(ENABLE_PEDANTIC_WARNINGS "Enable pedantic compiler warnings" On)
+option(ENABLE_TOOLS "Enable Umpire development tools" Off)
+
+if (ENABLE_DEVELOPER_DEFAULTS)
+  set(ENABLE_WARNINGS_AS_ERRORS On CACHE BOOL "")
+  set(ENABLE_TOOLS On CACHE BOOL "")
```

Comment:
I can't remember but will this set override anyone setting `ENABLE_TOOLS=Off` from the command line?

Reply:
Nope, it won't override a cache value, so you can do `-DENABLE_DEVELOPER_DEFAULTS=On -DENABLE_TOOLS=Off` just fine.